pkg/route/api/validation/validation.go

```diff
@@ -111,8 +115,94 @@ func ValidateRouteStatusUpdate(route *routeapi.Route, older *routeapi.Route) fie
 	return allErrs
 }
 
+type blockVerifierFunc func(block *pem.Block) error
+
+func publicKeyBlockVerifier(block *pem.Block) error {
```

Comment:
ummm... not sure? the uses I've seen for PEM headers in these types of blocks are for encrypted private keys, which we don't support